simplified-books-database/src/main/java/org/nypl/simplified/books/book_database/BookDRMInformationHandleACS.kt

```diff
@@ -0,0 +1,200 @@
+package org.nypl.simplified.books.book_database
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.nypl.drm.core.AdobeAdeptLoan
+import org.nypl.drm.core.AdobeLoanID
+import org.nypl.simplified.books.api.BookDRMInformation
+import org.nypl.simplified.books.api.BookDRMKind
+import org.nypl.simplified.books.book_database.api.BookDRMInformationHandle
+import org.nypl.simplified.books.book_database.api.BookFormats
+import org.nypl.simplified.files.FileUtilities
+import org.nypl.simplified.json.core.JSONParserUtilities
+import org.nypl.simplified.json.core.JSONSerializerUtilities
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.IOException
+import java.nio.ByteBuffer
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * An information handle for Adobe ACS DRM.
+ */
+
+class BookDRMInformationHandleACS(
+  private val directory: File,
+  format: BookFormats.BookFormatDefinition,
+  private val onUpdate: () -> Unit
+) : BookDRMInformationHandle.ACSHandle(), BookDRMInformationHandleBase {
+
+  private val objectMapper = ObjectMapper()
+  private val closed = AtomicBoolean(false)
+
+  companion object {
+    fun nameRights(format: BookFormats.BookFormatDefinition): String =
```

Comment:
You can omit the explicit return type for these two functions to match the functions below.